DataHub/Integration/01_ManageAttributes.md

```diff
@@ -0,0 +1,219 @@
+
+4. Press **F5** to initialize the *Core1 Platform* and to apply the changes.   
+
+
+
+## Delete an attribute
+
+You can move an attribute to the *DELETED ATTRIBUTES* tab if it is no longer needed.
+As there are usually dependencies on an attribute, for example through an attribute set or created products, it is strongly recommended not to delete an attribute.
+Instead, you can deactivate an attribute and thus prevent its use, see [Deactivate an attribute](#deactivate-an-attribute).
+
+#### Prerequisites
+
+At least one attribute has been created, see [Create an attribute](#create-an-attribute).
+
+> [Info] By default, a certain number of attributes has been predefined in the *PIM Basic Set* attribute set.
+
+#### Procedure
+
+*DataHub > Settings > Tab ATTRIBUTES*
+
+![Attributes](../../Assets/Screenshots/DataHub/Settings/Attributes/Attributes.png "[Attributes]")
+
+1. Select the checkbox of the attribute you want to delete in the list of attributes.   
+  The editing toolbar is displayed above the attributes list.
+
+2. Click the ![Delete](../../Assets/Icons/Trash03.png "[Delete]") (Delete) button in the toolbar. The *Confirm deletion of attribute?* window is displayed.
+
+  > [Info] In the window all dependencies to attribute sets, ETL-mappings and entity entries are displayed. Be aware that problems may occur if you delete an attribute with existing dependencies.
+
+3. Click the [CONFIRM] button in the bottom right corner.   
+  The attribute has been deleted and is moved to the *DELETED ATTRIBUTES* tab in the *DataHub* module. The *Confirm deletion of attribute?* window is closed.
+
+> [Info] The attribute is moved to the *DELETED ATTRIBUTES* tab, but it can still be recovered. To irretrievably delete the attribute, see [Finally delete an attribute](#finally-delete-an-attribute).
+
+
+
+## Recover an attribute
+
+If you have moved an attribute to the *DELETED ATTRIBUTES* tab, you can recover the attribute and make the deletion undone.
+
+#### Prerequisites
+
+At least one attribute has been moved to the *DELETED ATTRIBUTES* tab, see [Delete an attribute](#delete-an-attribute).
+
+#### Procedure
+
+*DataHub > Settings > Tab DELETED ATTRIBUTES*
+
+![Attributes](../../Assets/Screenshots/DataHub/Settings/DeletedAttributes/DeletedAttributes.png "[Attributes]")
+
+1. Select the checkbox of the attribute you want to recover in the list of deleted attributes.   
+  The editing toolbar is displayed above the deleted attributes list.
+
+2. Click the [RECOVER] button in the toolbar.   
+  The selected attribute has been recovered and is removed from the list of deleted attributes.
+
+[comment]: <> (Why can I not recover certain attributes? Is it a bug?)
+
+
+
+## Finally delete an attribute
+
+You can finally delete an attribute if it is no longer needed.
+As there are usually dependencies on an attribute, for example through an attribute set or created products, it is strongly recommended not to delete an attribute irretrievably.
```

Comment:
Vielleicht dazu: It is recommended to deactivate an attribute instead of deleting it to prevent any problems because of existing dependencies, see [Deactivate an attribute]...?